windows/hid.c

```diff
@@ -417,7 +417,7 @@ struct hid_device_info HID_API_EXPORT * HID_API_CALL hid_enumerate(unsigned shor
 		/* Check the VID/PID to see if we should add this
 		   device to the enumeration list. */
 		if ((vendor_id == 0x0 || attrib.VendorID == vendor_id) &&
-		    (product_id == 0x0 || attrib.ProductID == product_id)) {
+			(product_id == 0x0 || attrib.ProductID == product_id)) {
```

Comment:
IMHO "before" is better then "after"
1) Now part the condition is aligned with the code, which a con from readability POV
2) Before, the start of the condition in each line was aligned which is proc from readability POV

Reply:
Alright. Anyways current state when we have mixed tabs and spaces is a mess.

Can we stick to spaces-only (I prefer that) instead?

Reply:
historically HIDAPI is a tabs-only (except a few exceptions)